From 9952df17c02422173d11d772c6840e9be6080bae Mon Sep 17 00:00:00 2001
From: John Zhuge <jzhuge@cloudera.com>
Date: Thu, 11 Aug 2016 17:41:55 -0700
Subject: [PATCH 1733/2748] HDFS-10623. Remove unused import of
 httpclient.HttpConnection from TestWebHdfsTokens.
 (Contributed by Hanisha Koneru)

(cherry picked from commit c35c1e670367a60a68455807307520e12e68b834)

Change-Id: I9348b4e41447270fb38a1610423a14f930708d2a
---
 .../apache/hadoop/hdfs/web/TestWebHdfsTokens.java  |    1 -
 1 file changed, 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/web/TestWebHdfsTokens.java b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/web/TestWebHdfsTokens.java
index a7ef3e4..8a4a92c 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/web/TestWebHdfsTokens.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/web/TestWebHdfsTokens.java
@@ -36,7 +36,6 @@
 import java.security.PrivilegedExceptionAction;
 import java.util.Map;
 
-import org.apache.commons.httpclient.HttpConnection;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.FileUtil;
-- 
1.7.9.5

